kubernetes/linux/setup.sh

```diff
@@ -50,7 +50,7 @@ sudo apt-add-repository ppa:brightbox/ruby-ng -y
 sudo apt-get update
 sudo apt-get install ruby2.6 ruby2.6-dev gcc make -y
 # to fix CVE-2020-10663
-gem update json -v 2.5.1
+gem install json -v 2.5.1
```

Comment:
For CVE, we should upgrade to 2.5.1. I think, this makes both 2.1.0 & 2.5.1 co-exist which doesnt address cve.